employee_service/storage/employee.go

```diff
@@ -0,0 +1,44 @@
+package employee_storage
+
+import (
+	"HR-system/employee_service/models"
+
+	"gorm.io/gorm"
+)
+
+type Storage struct {
+	db *gorm.DB
+}
+
+func (s *Storage) SetDB(db *gorm.DB) {
+	s.db = db
+}
+
+func (s *Storage) AutoMigrate() error {
+	return s.db.AutoMigrate(&models.EmployeeModel{})
+}
+
+func (s *Storage) Create(value interface{}) *gorm.DB {
+	return s.db.Create(value)
+}
+
+func (s *Storage) DeleteById(id uint) *gorm.DB {
+	return s.db.Delete(&models.EmployeeModel{}, id)
+}
+
+func (s *Storage) GetById(id uint) ([]models.EmployeeModel, error) {
+	var employees []models.EmployeeModel
+	result := s.db.First(&employees, id)
```

Comment:
can we detect if the error is the NotFound or another issue and return the correct info?

Reply:
I will fix next PR